tests/unit_tests/test_wave_06.py

```diff
@@ -194,16 +194,18 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert not result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_d not in tai.inventory
+    assert item_e not in tai.inventory
+    assert item_f not in tai.inventory
+    assert item_a not in jesse.inventory
+    assert item_b not in jesse.inventory
+    assert item_c not in jesse.inventory
```

Comment:
I recommend swapping this to 'in' assertions rather than 'not in' assertions.  Asserting that the incorrect things are not in the inventory does not guarantee that the correct things are in the inventory.